x-pack/plugins/siem/public/resolver/view/defs.tsx

```diff
@@ -401,3 +358,63 @@ export const SymbolDefinitions = styled(SymbolDefinitionsComponent)`
   width: 0;
   height: 0;
 `;
+
+export const useResolverTheme = (): { colorMap: ColorMap; nodeAssets: NodeStyleMap } => {
+  const isDarkMode = useUiSetting<boolean>(DEFAULT_DARK_MODE);
+  const theme = isDarkMode ? euiThemeAmsterdamDark : euiThemeAmsterdamLight;
+
+  const getColor = (lightOption: string, darkOption: string): string => {
+    return isDarkMode ? darkOption : lightOption;
+  };
+
+  const colorMap = {
+    descriptionText: theme.euiColorDarkestShade,
+    full: theme.euiColorFullShade,
+    graphControls: theme.euiColorDarkestShade,
+    graphControlsBackground: theme.euiColorEmptyShade,
+    resolverBackground: theme.euiColorEmptyShade,
+    resolverEdge: getColor(theme.euiColorLightestShade, theme.euiColorDarkestShade),
+    resolverEdgeText: getColor(theme.euiColorDarkShade, theme.euiColorLightShade),
+  };
+
+  const nodeAssets = {
```

Comment:
:grey_question: As an SVG authoring practice, it seems a little strange to include i18n'd strings and stuff in a component called `<defs>` (although, to be fair, we were exporting pallette-y things that had nothing to do with paint servers or reusable <symbol>s before. Maybe we could call it `assets` or something now.